### PR TITLE
Update getting started and contributing

### LINKED ISSUE #85: Update the documentation
`README.md`, `CONTRIBUTING.md` and `LICENSE` all need updating:

- [x] Update website links to https://sketchbookjs.com
- [x] Update demo links to https://demo.sketchbookjs.com
- [x] Updated screenshot
- [ ] Update getting started instructions
- [x] Add ColonyDB sponsorship badge to README (with a link to `https://colonydb.com/?utm_source=sketchbook-js&utm_campaign=oss`)
- [x] Replace Spectrum links with GitHub discussion
- [x] Copyright year needs bumping
- [x] Add Michael to LICENSE
- [ ] Update GitHub project link

### PR DESCRIPTION
**Pull request by Mxchaeltrxn**
Closes #85

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,14 +35,16 @@ https://demo.sketchbookjs.com
 3.  Run on `http://localhost:3000/`:
 
     ```
-    yarn sketchbook start
+    yarn sketchbook start <path/to/design/file.json>
```
Comment (haydn) [inline README.md:38]: I think it'd be better to just use the generated file here:

```suggestion
    yarn sketchbook start sketchbook/example.json
```

That way someone can just copy and paste the commands from each step and get something up and running without having to think about it too much.

**Review by haydn — CHANGES_REQUESTED** [review]
@Mxchaeltrxn Sorry for being slow slack with this PR! One minor change.

File: README.md (hunk 1 above)
Comment (Mxchaeltrxn) [reply README.md]: Fixed @haydn

File: README.md [hunk 2]
```diff
@@ -35,14 +35,14 @@ https://demo.sketchbookjs.com
 3.  Run on `http://localhost:3000/`:
 
     ```
-    yarn sketchbook start
+    yarn sketchbook start example.json
```
Comment (haydn) [inline README.md:38]: This is incorrect. In the previous step the file gets generated at `sketchbook/example.json`:

```suggestion
    yarn sketchbook start sketchbook/example.json
```

Comment (Mxchaeltrxn) [reply README.md]: Sorry this was clumsy. I've fixed it @haydn

**Review by haydn — APPROVED** [review]

**Pull request merged**